README.md

```diff
@@ -68,6 +81,14 @@ Each database can belong to only one HA group.
 * **DomainCredential**: Domain credential could get list of cluster nodes.
 * **SqlAdministratorCredential**: SQL Server Administrator credential .
 
+### xSqlAlias
+
+* **SQLServerName**: The name of Alias (e.g. svr01\inst01).
+* **ServerName**: The name of real server.
+* **Protocol**: The protocol of either tcp or np (named pipes).
+* **RetryCount**: Maximum number of retries to check HA group existency.
+* **TCPPort**: The tcp port of the instance.
+
 ## Versions
 
```

Comment:
Thanks
I have added the note to the readme.md based on what I could see in other DSC resources. If there is anything else please let me know